gradle/versions.gradle

```diff
@@ -6,7 +6,8 @@ ext {
     compile_sdk_version = 28
     build_tools_version = '28.0.3'
 
-    support_lib = '27.1.1'
+    android_x_annotation = '1.0.2'
```

Comment:
Save just one variable `android_x`?

Reply:
I'm not sure that they're compatible [https://developer.android.com/jetpack/androidx/releases](url) Let's keep separate versions for now

Reply:
OK